log/json/logger.go

```diff
@@ -6,11 +6,13 @@ import (
 	"github.com/tidepool-org/platform/log"
 )
 
-func NewLogger(writer io.Writer, levels log.Levels, level log.Level) (log.Logger, error) {
+// CONCURRENCY: SAFE IFF writer is safe
```

Comment:
Did you mean `SAFE IFF`  as in "If and only if" ??

Reply:
Yes. IFF = if and only if